testfixtures/comparison.py

```diff
@@ -685,3 +685,27 @@ def diff(x, y, x_label='', y_label=''):
             y_label or 'second',
             lineterm='')
     )
+
+
+class RangeComparison:
+    """
+    An object that can be used in comparisons of numerics to validate
+    actual falls within a certain range
+    """
+    def __init__(self, lower_bound, upper_bound):
+        self.lower_bound = lower_bound
+        self.upper_bound = upper_bound
+
+    def __eq__(self, other):
+        try:
+            0 + other
```

Comment:
Seems odd. Why are you testing that addition is supported if the operator you actually need is `<=`?

Why not put the `try/except` around the actual comparison instead?